robots/BALANC3R/segway.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python3
```

Comment:
This file looks like a duplicate of GyroBalancer.py?

Reply:
If we chop this we can also chop parameters.py....but maybe I am missing something on why we needs these

Reply:
I've removed both segway.py and parameters.py.